src/main/java/lotto/domain/model/LottoGenerator.java

```diff
@@ -0,0 +1,8 @@
+package lotto.domain.model;
+
+import lotto.domain.business.LottoBuy;
+
+public interface LottoGenerator {
+    
+    LottoBuy generate();
```

Comment:
```suggestion
    LottoTickets generate();
```
LottoBuy를 생성해 반환하다보니 Auto인 경우에도 Manual, Manual인 경우에도 Auto 값을 가짐
각각의 Generator 구현체에서 바로 LottoTickets를 생성하도록 접근해 본다.
현재 구현과 어떻게 다른지 고민해 본다.

Reply:
`LottoBuy` 로 생성 후 반환하려고 하다 보니 LottoBuy의 생성자 틀에 억지로 맞추어야했다.
그래서 Manual 인 경우에도 Auto 값을 가지게 되었다
그에 맞춰 각각 구현체의 필드도 모두 같아야한다고 생각하게 되었다.